generator/src/main/java/com/scottlogic/deg/generator/outputs/targets/StreamOutputTarget.java

```diff
@@ -0,0 +1,23 @@
+package com.scottlogic.deg.generator.outputs.targets;
+
+import com.google.inject.Inject;
+import com.scottlogic.deg.generator.ProfileFields;
+import com.scottlogic.deg.generator.outputs.GeneratedObject;
+import com.scottlogic.deg.generator.outputs.datasetwriters.RowOutputFormatter;
+
+import java.util.stream.Stream;
+
+public class StreamOutputTarget implements OutputTarget {
+
+    private final RowOutputFormatter formatter;
+
+    @Inject
+    public StreamOutputTarget(RowOutputFormatter formatter) {
+        this.formatter = formatter;
+    }
+
+    @Override
+    public void outputDataset(Stream<GeneratedObject> generatedObjects, ProfileFields profileFields) {
+        generatedObjects.forEach(row -> System.out.println(formatter.format(row)));
```

Comment:
I'd suggest the following amends:
- Pass the stream into this type as a dependency; then this type makes no decisions as to where the data is written, only that it is written to a stream.
   - IoC can decide to provide `System.out` as appropriate
- Call `System.out.print` (or `stream.print`) instead of `println`, leave the return-line semantics to the formatter (same as @willsalt-sl 's comment)
- Consider whether there is a way to refactor/restructure the area so that CsvOutputTarget and JsonOutputTarget don't exist - at least in their current form - and instead are 'factories' for a stream that can be passed into this type by IoC